sssd_test_framework/utils/tools.py

```diff
@@ -599,27 +599,31 @@ def __init__(self, host: MultihostHost) -> None:
         """
         super().__init__(host)
 
-    def passwd(self, name: str | int) -> PasswdEntry | None:
+    def passwd(self, name: str | int, service="") -> PasswdEntry | None:
```

Comment:
Keep the default value as None, it will be neater, that is `service: str | None = None`